rpc/core/types/responses.go

```diff
@@ -60,11 +60,13 @@ type ResultCommit struct {
 	CanonicalCommit    bool `json:"canonical"`
 }
 
-// ResultTxStatus contains info to locate a tx in a committed block.
+// ResultTxStatus represents the status of a transaction during its life cycle.
```

Comment:
Does `ResultTxStatus` exist if the transaction isn't included in a block yet? "During its life cycle" makes me think this struct is used before + after it gets included in a block. If it's only after, perhaps:

```suggestion
// ResultTxStatus represents the status of a transaction after it is included in a block.
```

Reply:
ResultTxStatus also includes Status which tracks a transaction when it's pending in the mempool